core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -83,30 +87,30 @@ angular.module('oppia').component('contributionsAndReview', {
           color: '#e76c8c'
         }
       };
-      var SUGGESTION_TYPE_QUESTION = 'add_question';
-      var SUGGESTION_TYPE_TRANSLATE = 'translate_content';
+      ctrl.SUGGESTION_TYPE_QUESTION = 'add_question';
+      ctrl.SUGGESTION_TYPE_TRANSLATE = 'translate_content';
```

Comment:
Any particular reason for this change?

Reply:
Originally it was because we needed to access this for the topic selector in contributions-and-review.html, however since we moved the topic selector to the header we no longer need this and i have reverted this change